c/tskit/genotypes.c

```diff
@@ -333,7 +330,7 @@ tsk_vargen_update_genotypes_i8_sample_list(
             index = list_next[index];
         }
     }
-out:
+
```

Comment:
These functions can no long error, so we should make them void. Normally wouldn't bother, but this is perf sensitve.

Reply:
The return value appears to be a count of the number of missing genotypes, which is used to set the `has_missing_data` flag.

Reply:
Ah - my bad.